source/FreeRTOS_UDP_IP.c

```diff
@@ -310,7 +310,7 @@ BaseType_t xProcessReceivedUDPPacket( NetworkBufferDescriptor_t * pxNetworkBuffe
 {
     BaseType_t xReturn = pdPASS;
     FreeRTOS_Socket_t * pxSocket;
-    const UDPPacket_t * pxUDPPacket;
+    UDPPacket_t * pxUDPPacket;
```

Comment:
Was `const` causing an issue?

Reply:
Used that cool tool you showed me (https://cdecl.org/). I was reading the statement incorrectly....so no, the const shouldn't be causing an issue. I'll revert.

Reply:
Thanks!